packages/widget/src/stores/form/types.ts

```diff
@@ -0,0 +1,134 @@
+import type { UseBoundStoreWithEqualityFn } from 'zustand/esm/traditional';
+import type { StoreApi } from 'zustand/esm';
```

Comment:
I think ESM is used by default, so we can safely import from `zustand` and `zustand/traditional`.

Reply:
I'll check those and change them